Utils/src/cz/fidentis/landmarkParser/FPparser.java

```diff
@@ -80,8 +85,8 @@ public static FpModel load(String filePath) {
                 }
             }
         } catch (IOException | NumberFormatException | ParserConfigurationException | SAXException e) {
+            System.err.println(e);
```

Comment:
oh right, to som tam hodila len kvoli debugu, zabudla som to vyhodit :) prepisem, dobry postreh